quickstart/docker-compose.seq-ui.yaml

```
@@ -0,0 +1,45 @@
+services:
+  seq-ui:
+    image: ghcr.io/ozontech/seq-ui:latest
+    volumes:
+      - ./config.seq-ui.yaml:/seq-ui/config.yaml
+    ports:
+      - "5555:5555" # Default HTTP port
+      - "5556:5556" # Default gRPC port
+      - "5557:5557" # Default debug port
+    command: --config config.yaml
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)